geps/overview.md

```diff
@@ -112,6 +117,15 @@ What is out of scope: see [text from KEP][kep-when-to-use]. Examples:
     key goal of GEPs is to show why we made a decision and which alternatives
     were considered. If separate docs are used, it's important that we can
     still see all relevant context and decisions in the final GEP.
+* Q: When should I mark a GEP as `Prototyping` as opposed to `Provisional`?
+  * A: The `Prototyping` status carries the same base meaning as `Provisional`
+    in that consensus is not complete between stakeholders and we're not ready
+    to move toward releasing content yet. You should use `Prototyping` to
+    indicate to your fellow community members that we're in a state of active
+    practical tests and experiments which are intended to help us learn and
+    iterate on the GEP. These can include distributing content, but not under
+    any release (e.g. it has to be unavailable except when explicitly opted
+    in to).
```

Comment:
I think it's safer to exclude releases altogether unless we need them for some reason.
```suggestion
        any release channel.
```